<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BudgetDialog</class>
 <widget class="QDialog" name="BudgetDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>367</width>
    <height>179</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Budget</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QVBoxLayout" name="contentLayout">
     <item>
      <layout class="QHBoxLayout" name="initBudgetLayout">
       <item>
        <widget class="QLabel" name="initBudgetTitleLabel">
         <property name="text">
          <string>Budget Initial</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDoubleSpinBox" name="initBudgetButtonBox">
         <property name="suffix">
          <string>€</string>
         </property>
         <property name="maximum">
          <double>1000000.000000000000000</double>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="totalExpenLayout">
       <item>
        <widget class="QLabel" name="totalExpenTitleLabel">
         <property name="text">
          <string>Dépenses Totales</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="totalExpenValueLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="remainBudgetLayout">
       <item>
        <widget class="QLabel" name="remainBudgetTitleLabel">
         <property name="text">
          <string>Budget Restant</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="remainBudgetValueLabel">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="dialogButtonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>accepted()</signal>
   <receiver>BudgetDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>dialogButtonBox</sender>
   <signal>rejected()</signal>
   <receiver>BudgetDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
